### ISSUE TITLE
Feature request: Ephemeral tabs

### ISSUE THREAD

**Issue by jurijsk** [notepad-plus-plus/notepad-plus-plus#14594]
I would love to be able to use Notepad++ for the text I do not have intentions to store.

I found myself using a notepad as a temporary place to keep the text, compose comments, or to transform it using regex, or using it to change casing, and so on. Plenty of uses. And I do not think that I actually store any files that often. As a result, after some time, I have to do "purging" sessions where I try to recall what this bit of text about and if I should keep it. In 98% of cases, I close it without saving.

The feature I have in mind is either:
**A:** a shortcut to create a tab which will not ask for confirmation on closing, and if I close Notepad++ the tabs will be removed without the trace.
**B:** or a shortcut to close the tab without confirmation.

**Comment by alankilborn** [comment]
Your `B` can be achieved with a PythonScript:

```
if notepad.getCurrentFilename().startswith('new '):
    editor.setSavePoint()
    notepad.close()
```

**Comment by jurijsk** [comment]
@alankilborn  Thank your for the hint. I will try that and see if it sticks.

For those who want to try this. 
1. You have to install PythonScript script plugin. It is available from Plugins Admin. Not sure how it will behave if Python is not yet installed on your computer, see doc for that.
 2. Create the script with the code provided by alankilborn. Now you can invoke the script from Plugins menu.
 3. If you want to assign the shortcut to the script, first add your newly created script to Menu Items in PythonScript configuration, restart notepad++ and then go to `Settings -> Shortcut Mapper...` to finish the process.

**Comment by alankilborn** [comment]
>For those who want to try this.

Yes...sorry... I should have provided this link:  https://community.notepad-plus-plus.org/topic/23039/faq-desk-how-to-install-and-run-a-script-in-pythonscript

**Comment by Gilgamech** [comment]
I use the AutoSave plugin to achieve something similar:

- Configure AutoSave plugin to save to a specific folder. Feel free to enable security measures on that folder.
- Create new tabs for ephemeral uses. 
  - Open as many as you need. 
  - Let them keep their default names "New 1", "New 2"..."New N" etc.
- Let AutoSave plugin save these to the specific folder with the default names.
  - If there's a system issue, or if you need to reboot for Windows Updates etc, the data is already on disk and the file can just be reopened.
- When you don't need a file or data anymore, close the tab.
  - If the universe changes and you suddenly need the data again, just un-close the tab `Ctrl+Shift+T`. 
  - Long-term, the file will eventually probably get overwritten. 
    - The first "New 1" will get overwritten probably 80% to 90% as much as "New 2", which will probably be 80% to 90% more often than "New 3" etc. 

If you don't open many of these at the same time, you won't really need to garbage collect. My use patterns only create 4-5 of these files.
